### PR TITLE
Add IP addresses to hub logs

### PR DESCRIPTION
**Pull request by lukemelia**

### CODE REVIEW

**Review by backspace — CHANGES_REQUESTED** [review]
I deployed to staging this because I had a suspicion it might not work, in previous experiences like this I’ve found that it logs ≈internal IPs (load balancer?) and sadly that’s the case here:

image

Surely there’s a way to get the external IP 🤔

**Comment by lukemelia** [comment]
Seems like AWS ELB may handle the relevant header a bit unusually. https://github.com/koajs/koa/issues/1094

**Comment by lukemelia** [comment]
Thanks @backspace. I've confirmed a fix on staging:

```
2022-05-19T19:10:15.428Z YNQS2A: hub/server start [74.66.23.130] GET /status
2022-05-19T19:10:15.429Z YNQS2A: hub/server finish [74.66.23.130] GET /status 404
2022-05-19T19:10:25.646Z YNQS2A: hub/server start [::ffff:10.91.2.150] GET /
2022-05-19T19:10:25.647Z YNQS2A: hub/server finish [::ffff:10.91.2.150] GET / 200
```

The first request is external and the second request is from the internal health checker.

File: yarn.lock [hunk 1]
```diff
@@ -10606,7 +10606,7 @@ aws4@^1.8.0:
   resolved "https://registry.yarnpkg.com/aws4/-/aws4-1.8.0.tgz#f0e003d9ca9e7f59c7a508945d7b2ef9a04a542f"
   integrity sha512-ReZxvNHIOv88FlT7rxcXIIC0fPt4KZqZbOlivyWtXLt8ESx84zd3kMC6iK5jVeS2qt+g7ftS7ye4fi06X5rtRQ==
 
-axe-core@^4.1.4:
+axe-core@4.1.4, axe-core@^4.1.4:
```
Comment (backspace) [inline yarn.lock:10609]: This (and the below) is constantly happening so it’s not something you did but I’d love to figure out the cause 🧐

**Review by backspace — APPROVED** [review]
Excellent!

**Pull request merged**